src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -7391,6 +7391,249 @@ namespace SimAirServingZones {
         return ReheatCoilInHumRatForSizing;
     }
 
+    void CheckWaterCoilIsOnAirLoop(int const CompTypeNum, std::string const CompType, std::string const CompName, bool &WaterCoilOnAirLoop)
+    {
+
+        // PURPOSE OF THIS FUNCTION:
+        // This function returns true if a water coil that has water controller is either on
+        // primary air or outdoor air system branch. Searches for water coil name and type
+        // that match components list in primary air and outside air systems.
+
+        // METHODOLOGY EMPLOYED:
```

Comment:
I know this is what we used to do with these functions but we don't really need all these empty comment blocks.  If everything else is OK here I won't let that hold it up, but in the future clean them out.

Reply:
Removed empty blocks as suggested.